specificatie/gba-genereervariant/openapi.json

```diff
@@ -1378,6 +1378,24 @@
             "$ref" : "#/components/schemas/GbaDatum"
           }
         }
+      },
+      "GbaKindNaamBasis" : {
```

Comment:
Waarom los je het niet op in GbaNaamBasis? Den heb je in één keer zowel voor ouders, partners als kinderen het probleem opgelost.

Reply:
Omdat ik er vanuit ging dat dat juist niet de bedoeling was. Ik maak uit je reactie echter op dat dit geen probleem is dus dan is dat natuurlijk de juiste wijze.